draft-ietf-quic-http.md

```diff
@@ -1207,7 +1207,14 @@ the requests if the requested resource is already being pushed.
 When a server later fulfills a promise, the server push response is conveyed on
 a push stream (see {{push-streams}}). The push stream identifies the Push ID of
 the promise that it fulfills, then contains a response to the promised request
-using the same format described for responses in {{request-response}}.
+using the same format described for responses in {{request-response}}. Due to
+reordering, data on a push stream can arrive before the corresponding
+PUSH_PROMISE, in which case both the associated client request and the pushed
+request headers are unknown. Clients which receive a new push stream with an
```

Comment:
CMOS change:
```suggestion
request headers are unknown. Clients that receive a new push stream with an
```

Reply:
That was based on identical text for DUPLICATE_PUSH a few lines up. I've now also changed that to keep consistency.

Reply:
You haven't changed it _here_, though.

Reply:
Sorry, wasn't sure if I should do this or the editors before merge. Should be accepted now.